cnf/central.mvn

```diff
@@ -113,6 +113,7 @@ org.osgi:org.osgi.service.coordinator:1.0.2
 org.osgi:org.osgi.service.event:1.4.0
 org.osgi:org.osgi.service.http.whiteboard:1.1.0
 org.osgi:org.osgi.service.http:1.2.1
+org.osgi:org.osgi.service.jaxrs:jar:1.0.0
```

Comment:
I never expected to ever see Jax rs in a project I control :-) I hate it with a vengance :-)

Did you ever look at my OpenAPI project?

Reply:
years ago when i created an openapi from the jaxrsruntimedto